### PR TITLE
Make error messages better

### LINKED ISSUE #6605: Configuration error message contains wrong version
**Do you want to request a *feature* or report a *bug*?**
*bug*

**What is the current behavior?**
If there is a invalid property in configuration file the error contains wrong webpack version (2).

https://github.com/webpack/webpack/blob/6a5d081f294a225d9f02b56c55fdcd61b7b4db20/lib/WebpackOptionsValidationError.js#L158

https://github.com/webpack/webpack/blob/6a5d081f294a225d9f02b56c55fdcd61b7b4db20/lib/WebpackOptionsValidationError.js#L172

**If the current behavior is a bug, please provide the steps to reproduce.**
Run webpack with a invalid property inside configuration.

**What is the expected behavior?**
- Webpack version is `>= 2`
- Print current webpack version (I've create a PR with this option)

**If this is a feature request, what is motivation or use case for changing the behavior?**
N/A

**Please mention other relevant information such as the browser version, Node.js version, webpack version, and Operating System.**
N/A

### PR DESCRIPTION
**Pull request by kvrlk**
**What kind of change does this PR introduce?**

 bugfix

**Did you add tests for your changes?**
No, just edited already existing test.

**If relevant, link to documentation update:**

**Summary**

It should fix #6605
**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

File: lib/WebpackOptionsValidationError.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ class WebpackOptionsValidationError extends WebpackError {
 					baseMessage +
 					"\n" +
 					"For typos: please correct them.\n" +
-					"For loader options: webpack 2 no longer allows custom properties in configuration.\n" +
+					"For loader options: webpack since version 2 no longer allows custom properties in configuration.\n" +
```
Comment (michael-ciniawsky) [inline lib/WebpackOptionsValidationError.js]: `>= v2.0.0`

Comment (kvrlk) [reply lib/WebpackOptionsValidationError.js]: ok

File: lib/WebpackOptionsValidationError.js [hunk 2]
```diff
@@ -155,7 +155,7 @@ class WebpackOptionsValidationError extends WebpackError {
 					case "debug":
 						return (
 							`${baseMessage}\n` +
-							"The 'debug' property was removed in webpack 2.\n" +
+							"The 'debug' property was removed since webpack 2.\n" +
```
Comment (michael-ciniawsky) [inline lib/WebpackOptionsValidationError.js]: Here aswell please:

since`webpack >= v2.0.0`

Comment (probablyup) [reply lib/WebpackOptionsValidationError.js]: Not sure I agree with this particular change. It breaks the grammar of the sentence.

**Comment by probablyup** [comment]
Looks to be a dup of https://github.com/webpack/webpack/pull/6609

File: lib/WebpackOptionsValidationError.js (hunk 2 above)
Comment (kvrlk) [reply lib/WebpackOptionsValidationError.js]: So I need to just keep `` in ``?

**Comment by kvrlk** [comment]
@probablyup checked this. Looks useless, because it's not fixes actual problem. I'm ok with closing this if author of that PR would edit it.

**Comment by marcalexiei** [comment]
Hi,
I can edit my PR without problem.

L. 158 -> `The 'debug' property was removed from webpack since version 2.0.0.`
L. 172 -> `For loader options: webpack >= 2.0.0 no longer allows custom...`

Do you prefer this structure?

**Comment by kvrlk** [comment]
@marcalexiei see latest commit

**Comment by marcalexiei** [comment]
Changes applied to #6609 PR.

**Comment by sokra** [comment]
Thanks

**Pull request merged**